spec/datadog/core/environment/variable_helpers_spec.rb

```diff
@@ -85,11 +86,11 @@
         '',
         'false',
         'FALSE',
-        0,
-        1
+        '0',
```

Comment:
Can we add a couple of example with leading and trailing spaces? such as `'   1'`  and `'1   '`

Reply:
Certainly!